packaging/README.branch.md

```diff
@@ -0,0 +1,23 @@
+# Branch maintaining
+
+## The `master` branch
+
+The `master` branch contains features and bug fixes since last tagged release.
+
+## The `stable_v<x.y.z>` branch
```

Comment:
IMHO this document is missing when there will be a new stable_* branch, how long it will be supported and probably also when we will do a new release in general.

IMHO it would be better to start releasing x.y.0 releases from master, i.e. 0.1.0 next, have a 0.1.x branch and releases 0.1.1 etc. with bugfixes from it, if we want to have multiple released versions.

Currently the last release is from 25 July, which is quite a long time.

Reply:
In the standup meeting today @EdDev and me preferred a branch like `nmstate-0.1` once we release `0.1.1` from master with the next release `0.1.2` and changing master to `0.2.0`.